app/src/App.js

```diff
@@ -26,9 +27,29 @@ const moonPhaseUrl = `https://www.icalendar37.net/lunar/api/?lang=en&month=${
 //   console.log(data);
 //   return data;
 // };
-
+const calculateMoonPhase = (num) => {
```

Comment:
Wow the moon waxing/waning images are really cool! Nice moon phase calculation rendering!

![image](https://user-images.githubusercontent.com/76133327/118751801-d22eb600-b816-11eb-93bf-6dc3dc42bed5.png)